src/content/configuration/dev-server.mdx

```diff
@@ -964,7 +965,7 @@ module.exports = {
 
 `function (devServer)`
 
-Provides the ability to execute custom middleware prior to all other middleware
+Provides the ability to execute custom middleware before all other middleware
```

Comment:
What's wrong with `prior to`?

Reply:
Prior to has similar meaning to before ,you can check below.

![prior to - Google Search - Google Chrome 26-09-2021 8 59 08 AM](https://user-images.githubusercontent.com/72331432/134792397-732f2c10-78fe-4752-b5c9-4fef4a8123d9.png)

Reply:
Then let's stick with `prior to`.

Reply:
> Then let's stick with `prior to`.

so can you suggest me the changes or i should make a new Pullrequest ?

Reply:
```suggestion
Provides the ability to execute custom middleware prior to all other middleware
```